metadata-ingestion/adding-source.md

```diff
@@ -130,8 +130,8 @@ class FileSource(Source):
 
 #### 7.2 Write custom documentation
 
-- Create a copy of [`source-docs-template.md`](./source-docs-template.md) and edit all relevant components. 
-- Name the document as `<plugin.md>` and move it to `metadata-ingestion/docs/sources/<platform>/<plugin>.md`. For example for the Kafka platform, under the `kafka` plugin, move the document to `metadata-ingestion/docs/sources/kafka/kafka.md`.
+- Use the docstring section of the source class to add documentation. This section supports markdown formatting.
```

Comment:
Does the docstring section contain all of the relevant detail that was available in `source-docs-template.md` ?